bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadEntryProcessorV3.java

```diff
@@ -350,6 +344,43 @@ protected void registerEvent(boolean failed, OpStatsLogger statsLogger, Stopwatc
         }
     }
 
+    ByteString parseToByteString(ByteBuf entryBody) {
+        ByteString aggregated = null;
+        ByteString piece;
```

Comment:
Having this variable introduced at this level isn't needed.

Reply:
makes sense to me, fixed